src/components/CandidateServiceTile.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import AppIcon from 'cozy-ui/react/AppIcon'
+import AppLinker, { generateWebLink } from 'cozy-ui/react/AppLinker'
+import { withClient } from 'cozy-client'
+
+class CandidateServiceTile extends React.Component {
+  render() {
+    const { t, slug, name, client } = this.props
+    const { domain, secure } = this.context
```

Comment:
domain and secure can be deduced from the client. This way we do not have to touch the context manually.